pydal/migrator.py

```diff
@@ -429,7 +429,7 @@ def migrate_table(self, table, sql_fields, sql_fields_old, sql_fields_aux,
                     drop_expr % (table._rname, key_tmp)
                 ]
                 metadata_change = True
-            elif sql_fields[key]['type'] != sql_fields_old[key]['type']:
```

Comment:
No, we're migrating on every request because the actual difference in metadata does **NOT** trigger this condition. But it does trigger a call to `migrate_table()` in the parent migrator method. And I'm not dropping the condition. Look at the next line in the diff.